app/src/main/java/com/isc/hermes/database/collection/Incidents.java

```diff
@@ -0,0 +1,56 @@
+package com.isc.hermes.database.collection;
+
+import com.isc.hermes.database.Hermesdb;
+import com.mongodb.client.MongoCollection;
+
+import org.bson.Document;
+
+/**
+ * This class that represents the collection of incidents from the hermes database.
+ */
+public class Incidents implements Collectible {
+
+    private Hermesdb hermesdb;
+    private final String COLLECTION_NAME;
```

Comment:
This variable is final and is a constant, it should be initialized in the same declaration and not in the constructor. 
Example: `private final String COLLECTION_NAME = "incidents"`

Reply:
done